pagerduty/CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+# CHANGELOG - PagerDuty
+
```

Comment:
```suggestion
## 1.0.0 / 2021-10-20

* [Added] Initial release.
```

Add an initial entry similar to the suggestion above.

Reply:
done